### ISSUE TITLE
The playlist window automatically gets focus

### ISSUE THREAD

**Issue by DreamMaoMao** [smplayer-dev/smplayer#1049]
**Is your feature request related to a problem? Please describe.**
Currently, when we open a playlist with a shortcut, the playlist subwindow does not automatically get keyboard focus, so you cannot use the keyboard to select.

Although he can use the mouse to click the playlist window to get the keyboard focus, but the use of this logic is very strange, which makes the shortcut to lose its meaning (can not simply use the keyboard to complete the selection).

**Describe the solution you'd like**
The playlist sub-window automatically gets the keyboard focus when it pops up, and automatically resets the focus back to the main window when it closes。
